doc/build-freebsd.md

```diff
@@ -64,10 +64,11 @@ sh/bash: export BDB_PREFIX=[path displayed above]
 #### GUI Dependencies
 ###### Qt5
 
-Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, we need to install `qt5`. Skip if you don't intend to use the GUI.
+Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, we need to install the necessary parts of Qt. Skip if you don't intend to use the GUI.
 ```bash
-pkg install qt5
+pkg install qt5-buildtools qt5-core qt5-gui qt5-linguisttools qt5-network qt5-qmake qt5-testlib qt5-widgets
```

Comment:
1. Why `qt5-qmake`?

2. The build system (`build-aux/m4/bitcoin_qt.m4`) checks for the `qt5-network`  and `qt-dbus` packages, which both are dependencies of the `qt5-gui`. However, only `qt5-network` is listed explicitly. For consistency sake, I suggest to list either both packages or none.

Reply:
> 1. Why `qt5-qmake`?

This PR originated from https://github.com/hebasto/bitcoin/pull/133#discussion_r1551480291 where I was testing bitcoin+cmake on freebsd. `qt5-qmake` is needed because of this error:

```
CMake Error at /usr/local/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:14 (message):
  The imported target "Qt5::Core" references the file

     "/usr/local/lib/qt5/bin/qmake"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/usr/local/lib/cmake/Qt5Core/Qt5CoreConfigExtras.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /usr/local/lib/cmake/Qt5Core/Qt5CoreConfigExtras.cmake:9 (_qt5_Core_check_file_exists)
  /usr/local/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:232 (include)
  /usr/local/lib/cmake/Qt5/Qt5Config.cmake:28 (find_package)
  src/qt/CMakeLists.txt:48 (find_package)
```

I copied the list of packages into this PR which is against `master`/autotools, assuming that the dependencies are the same. However now I explicitly tested with `master`/autotools and it compiles without `qt5-qmake`, eh!?

Starting without any qt packages, here are the errors and the packages needed to resolve them:

```
checking for Qt5Core >= 5.11.3... no
configure: error: Qt5Core >= 5.11.3 not found
```
needed: `qt5-core` (no dependencies)

---

```
checking for Qt5Gui >= 5.11.3... no
configure: error: Qt5Gui >= 5.11.3 not found
```
needed: `qt5-gui` (dependencies that will be automatically installed: `qt5-dbus`, `qt5-network`)

---

```
checking for Qt5Widgets >= 5.11.3... no
configure: error: Qt5Widgets >= 5.11.3 not found
```
needed: `qt5-widgets` (no dependencies)

---

```
checking for Qt5Test >= 5.11.3... no
```
optional, non-mandatory: `qt5-testlib` (no dependencies)

---

```
checking for moc-qt5... no
checking for moc5... no
checking for moc... no
configure: error: MOC not found
```
needed: `qt5-buildtools` (no dependencies)

---

```
checking for lrelease-qt5... no
checking for lrelease5... no
checking for lrelease... no
configure: error: LRELEASE not found
```
needed: `qt5-linguisttools` (dependencies that will be automatically installed: `qt5-xml`).

---

Thus, I removed `qt5-network` and `qt5-qmake` from the list.